hack/common.sh

```diff
@@ -159,9 +184,16 @@ os::build::build_binaries() {
     for platform in "${platforms[@]}"; do
       os::build::set_platform_envs "${platform}"
       echo "++ Building go targets for ${platform}:" "${targets[@]}"
-      go install "${goflags[@]:+${goflags[@]}}" \
-          -ldflags "${version_ldflags}" \
-          "${binaries[@]}"
+      for artifact in ${binaries[@]}; do
```

Comment:
I'll say it again, if a strong case can be made for a wrapped kube e2e, we should do the upstream work to allow wrapping it. If a strong case can't be made, we should not wrap it :)